scripts/o3de/o3de/repo_properties.py

```diff
@@ -352,7 +392,13 @@ def add_parser_args(parser):
     group = parser.add_argument_group('properties', 'arguments for modifying individual remote repo properties.')
     group.add_argument('--repo-name','-rn',  type=str, required=False,
                        help='The name of the remote repository.')
-    group = parser.add_mutually_exclusive_group(required=False)
+    group.add_argument('--auto-update', '-au', type=str, nargs='*', required=False,
+                       help='Checks for any new Gems/Projects/Templates in the associated directories that have not been added'
+                       ' to your repo.json fields.'
+                       ' Pass in lower case object type as args that should look like this: gem/ project/ template'
```

Comment:
we don't really need the forward slashes to pass multiple args do we?
```suggestion
                       ' Optionally, provide the object types to update as args like this: --auto update gem project template'
```